openlibrary/plugins/openlibrary/js/plot_tooltip_graph.js

```diff
@@ -0,0 +1,64 @@
+export default function plot_tooltip_graph(node, data, tooltip_message) {
```

Comment:
Can we put these 2 together into a `plot_utils.js` file? I think they're related enough to be in one file.